### PR TITLE
Fix agent install problem

### PR DESCRIPTION
**Pull request by nnDarshan**
Fixes the issues related to installation of node-agent from source

tendrl-bug-id: Tendrl/node_agent#80

### CODE REVIEW

**Comment by nnDarshan** [comment]
@r0h4n @shtripat @nthomas-redhat Please review

File: doc/source/installation.rst [hunk 1]
```diff
@@ -32,11 +32,22 @@ bundeled for syslog and journald logging as well. These could be used similarly
...
+
+Clone and install the node-agent
+
     $ git clone https://github.com/Tendrl/node_agent.git
     $ cd node_agent
     $ workon node_agent
     $ pip install .
 
+One of the dependency of node-agent needs a higher version of setuptools, So user
+has to manually upgrade the setuptools package to alteast setuptools>=11.3
```
Comment (shtripat) [inline doc/source/installation.rst]: Is it for node_agent only? BTW which package need this?

Comment (nnDarshan) [inline doc/source/installation.rst]: yes, this latest version of setuptools is needed only for node_agent. One of the dependency of ansible has this dependency.

Comment (shtripat) [inline doc/source/installation.rst]: ok

Comment (r0h4n) [inline doc/source/installation.rst]: You can directly add to requirements.txt
setuptools>=11.3

File: doc/source/installation.rst [hunk 2]
```diff
@@ -32,11 +32,22 @@ bundeled for syslog and journald logging as well. These could be used similarly
...
+
+    $ pip install setuptools --upgrade
```
Comment (r0h4n) [inline doc/source/installation.rst]: Not needed here, just add to requirements.txt "setuptools>=11.3"

File: requirements.txt [hunk 3]
```diff
@@ -6,3 +6,4 @@ ansible
 python-etcd
 gevent>=1.0
 greenlet>=0.3.2
+setuptools>=11.3
```
Comment (r0h4n) [inline requirements.txt]: Please mention as a # comment as to why we need setuptools>=11.3

**Pull request merged**